rapier/rules-update-2020-07.md

```diff
@@ -0,0 +1,63 @@
+--- 
+title: Updates from latest Society Rules - July 2020
+layout: default 
+--- 
+ 
+* TOC 
+{:toc} 
+ 
+**Author**: Master Brian de Moray, KRM 
+ 
+**Last Modified**: {{ page.last_modified_at | date: '%Y-%m-%d' }}
+ 
+**Revision**: {{ site.git.last_commit.short_sha }}
+
+# Rubber-tipped Spears
+
+Atlantia now allows for the use of rubber-tipped spears in both rapier and C&T combat.  Rapier Spears are specifically precluded from use in light rapier.  For the foreseeable future, all Rapier Spear activities must be actively marshaled by a rapier marshal with a Rapier Spear authorization.

```

Comment:
Definition of "actively marshaled"?

Reply:
All of our activities require an authorized Marshall to oversee them. This language of this call out suggests that some how rapier with spear is different.  Perhaps we could highlight that this is a common requirement.  "As with all Marshall activities, activities involving rapier and spear must be marshaled by a marshal authorized in all of the involved forms of combat. For example, rapier combat involving my C&T and spears must be marshaled by a rapier marshal authorized in both C&T as well as spear."

Reply:
My intent is to move towards aligning with the armored community regarding authorizations and marshal warrants.  In the armored authorizations, only one of the overseeing marshals must be authorized in the weapons form being authorized.  Additionally, an armored marshal is not required to be authorized a form of armored combat to oversee it.

That said, rapier spears are called out as different on purpose.  My concern is that rapier spear combat needs to be overseen by someone with experience in rapier spear, regardless if it is used in rapier or C&T.

Reply:
feedback addressed in updates